chalicelib/checks/es_checks.py

```diff
@@ -1,4 +1,5 @@
 import time
+import datetime
```

Comment:
I see you newly importing this but I don't see where you used it.

Reply:
If you look at this file you'll see `datetime` is referenced yet it is not imported. I'm guessing when this is packaged it gets it some other way, as I believe the checks work. I added the import so it's more clear.